edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/TwinConfigSource.cs

```diff
@@ -22,21 +22,28 @@ public class TwinConfigSource : IConfigSource
         readonly AsyncLock configLock = new AsyncLock();
         readonly ITwinManager twinManager;
         readonly string id;
-        readonly RouteFactory routeFactory;
+        readonly EdgeHubConfigParser configParser;
         readonly Core.IMessageConverter<TwinCollection> twinCollectionMessageConverter;
         readonly Core.IMessageConverter<Twin> twinMessageConverter;
         readonly VersionInfo versionInfo;
         readonly EdgeHubConnection edgeHubConnection;
         Option<TwinCollection> lastDesiredProperties;
 
-        public TwinConfigSource(EdgeHubConnection edgeHubConnection, string id, VersionInfo versionInfo, ITwinManager twinManager, Core.IMessageConverter<Twin> messageConverter, Core.IMessageConverter<TwinCollection> twinCollectionMessageConverter, RouteFactory routeFactory)
+        public TwinConfigSource(
+            EdgeHubConnection edgeHubConnection,
+            string id,
+            VersionInfo versionInfo,
+            ITwinManager twinManager,
+            Core.IMessageConverter<Twin> messageConverter,
+            Core.IMessageConverter<TwinCollection> twinCollectionMessageConverter,
+            EdgeHubConfigParser configParser)
```

Comment:
I did some minor refactoring to remove `TwinConfigSource` static dependency on `EdgeHubConfigParser` for better testability.